### PR TITLE
Data fusion no. 99999

### PR DESCRIPTION
**Pull request by dsonyy**

### CODE REVIEW

File: attitude-estimator/attitude_estimator.cpp [hunk 1]
```diff
@@ -0,0 +1,945 @@
...
+	// Update the alternative attitude estimate representations
+	updateEuler(); // Resets m_Ehat and m_eulerValid internally
+	updateFused(); // Resets m_Fhat, m_FhatHemi and m_fusedValid internally
+
+	// Initialise the gyro bias estimate
+	if(resetGyroBias)
+		setGyroBias(0.0, 0.0, 0.0); // Resets m_bhat internally
+
+	// Initialise the remaining size 3 internal variables
+	for(i = 0; i < 3; i++)
```
Comment (bartekpacia) [inline attitude-estimator/attitude_estimator.cpp:73]: lmao that;s some clever shit

Comment (dsonyy) [reply attitude-estimator/attitude_estimator.cpp:73]: yeah, but I'm too dumb to fully understand the code and the math behind it. Btw the original code is from there: https://github.com/AIS-Bonn/attitude_estimator

**Comment by bartekpacia** [comment]
hello darkness my old friend @dsonyy

**Comment by dsonyy** [comment]
here we go again

**Pull request merged**